linode/manager
# React Query for Volumes - Linode Storage

## Description 📝

- Makes `http://localhost:3000/linodes/:id/storage` use React Query

## Preview 📷

https://user-images.githubusercontent.com/115251059/198066432-616af01d-d604-47a2-b6d3-2be813113bfd.mov

## How to test 🧪

- Test all possible volume actions from `http://localhost:3000/linodes/:id/storage` 
  - When testing, check back with the `http://localhost:3000/volumes` page to make sure the cache is updating properly as mutations occur
- Test volume actions `http://localhost:3000/volumes` and make sure `http://localhost:3000/linodes/:id/storage` remains up to date

**How do I run relevant unit or e2e tests?**
